form/config.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+//check if environment is local or prod
+if (stristr($_SERVER['HTTP_HOST'], 'local') || (substr($_SERVER['HTTP_HOST'], 0, 7) == '192.168')) {
+    //db connection
+    $dbHost = "localhost";
+    $dbName = "ballers_new";
+    $dbUser = "root";
+    $dbPass = "";
+    $mysqli = mysqli_connect($dbHost, $dbUser, $dbPass, $dbName);
+    
+    if (!$mysqli) {
```

Comment:
Repeated Code: I believe that this line can be moved outside the if statement block instead of writing it here to avoid repeating in the prod block below.